### PR TITLE
AVerMedia.AssistCentral version 1.2.0.13

### PR DESCRIPTION
**Pull request by infinitepower18**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69482)

### CODE REVIEW

File: manifests/a/AVerMedia/AssistCentral/1.2.0.13/AVerMedia.AssistCentral.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.AssistCentral
+PackageVersion: 1.2.0.13
+PackageLocale: en-US
+Publisher: AVerMedia TECHNOLOGIES, Inc.
+PackageName: AVerMedia Assist Central
+License: © 2022 AVerMedia Technologies, Inc.
```
Comment (Trenly) [inline manifests/a/AVerMedia/AssistCentral/1.2.0.13/AVerMedia.AssistCentral.locale.en-US.yaml:9]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

File: manifests/a/AVerMedia/AssistCentral/1.2.0.13/AVerMedia.AssistCentral.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.AssistCentral
+PackageVersion: 1.2.0.13
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
```
Comment (Trenly) [inline manifests/a/AVerMedia/AssistCentral/1.2.0.13/AVerMedia.AssistCentral.installer.yaml:8]: Installer does not install silently in a sandbox environment; Silent switches must be provided for the package

File: manifests/a/AVerMedia/AssistCentral/1.2.0.13/AVerMedia.AssistCentral.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.AssistCentral
+PackageVersion: 1.2.0.13
+PackageLocale: en-US
+Publisher: AVerMedia TECHNOLOGIES, Inc.
+PackageName: AVerMedia Assist Central
+License: © 2022 AVerMedia Technologies, Inc.
+Copyright: Copyright (c) AVerMedia
+ShortDescription: AVerMedia Assist Central
```
Comment (Trenly) [inline manifests/a/AVerMedia/AssistCentral/1.2.0.13/AVerMedia.AssistCentral.locale.en-US.yaml:11]: Please provide a better description of what the application is used for

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by infinitepower18** [comment]
Closing this, doesn't seem to be able to install silently with their custom installer

**Comment by krystofkrticka** [comment]
You could try to manually edit manifest to use schema 1.1 instead of 1.2 if it is not portable application or feature introduced in schema 1.2.

**Pull request closed without merging**